action.yml

```
@@ -15,6 +15,10 @@ inputs:
     description: Whether linters should try to fix code style issues automatically
     required: false
     default: "false"
+  skip_verification:
+    description: 'Appends --no-verify to commit and push actions'
```

```suggestion
  git_no_verify:
    description: Bypass the pre-commit and pre-push git hooks
```

For clarity I'd  name it `git_no_verify`. What do you think?

---

@ocean90 LGTM! Committed suggestion via UI and updated other code via followup commit